fetch/api/basic/request-upload.any.js

```diff
@@ -123,3 +125,27 @@ testUploadFailure("Fetch with POST with ReadableStream containing Blob", url,
       controller.close();
     }})
   });
+
+promise_test(async (test) => {
+  const resp = await fetch(
+    "/fetch/connection-pool/resources/network-partition-key.py?"
+    + `status=421&uuid=${token()}&partition_id=${get_host_info().ORIGIN}`
+    + `&dispatch=check_partition&addcounter=true`,
+    {method: "POST", body: "foobar"});
+  assert_equals(resp.status, 421);
+  const text = await resp.text();
+  assert_equals(text, "ok21");
```

Comment:
Why is two requests and one connection sufficient? This could maybe also do with a comment explaining this.

Reply:
Ditto.